### ISSUE TITLE
Runtime error on v1.1.0

### ISSUE THREAD

**Issue by mununki** [zth/rescript-relay#441]
When I try to v1.1.0 on my react-native project then I encounter the runtime error as below;

```
record.getType is not a function. (In 'record.getType()', 'record.getType' is undefined)

in source RescriptRelay.bs.js (105:67)
```

v1.0.4 and v1.0.5 seem fine.

**Comment by zth** [comment]
Did you upgrade Relay to `15.0.0`?

**Comment by mununki** [comment]
> Did you upgrade Relay to `15.0.0`?

Ah! I missed it. Let me try again with Relay `15.0.0`.
